pkg/asset/manifests/ingress_test.go

```diff
@@ -187,14 +187,14 @@ func TestGenerateIngerssDefaultPlacement(t *testing.T) {
 			name:                      "none-platform single node with 0 or 1 day-1 workers",
 			installConfigBuildOptions: []icOption{icBuild.forNone()},
 			controlPlaneTopology:      configv1.SingleReplicaTopologyMode,
-			infrastructureTopology:    configv1.SingleReplicaTopologyMode,
+			infrastructureTopology:    configv1.HighlyAvailableTopologyMode,
 			expectedIngressPlacement:  configv1.DefaultPlacementControlPlane,
 		},
 		{
 			name:                      "none-platform multi-node with 1 day-1 worker",
 			installConfigBuildOptions: []icOption{icBuild.forNone()},
 			controlPlaneTopology:      configv1.HighlyAvailableTopologyMode,
-			infrastructureTopology:    configv1.SingleReplicaTopologyMode,
+			infrastructureTopology:    configv1.HighlyAvailableTopologyMode,
```

Comment:
When there is a single control-plane node, we should end up with `controlPlaneTopology==configv1.SingleReplicaTopologyMode` and `infrastructureTopology==configv1.SingleReplicaTopologyMode`. I don't see a test for that.